numpy/core/tests/test_multiarray.py

```diff
@@ -2907,7 +2907,8 @@ def norm(result):
                                      "__array_ufunc__")
                     else:
                         if (isinstance(obj, np.ndarray) and
-                                not hasattr(obj, "__array_ufunc__")):
+                            (obj.__class__.__array_ufunc__ is
```

Comment:
I need to look on the class since the instance has a wrapped version (which calls the class version with `self` as first argument), so one cannot compare anymore. 

Here, `obj.__class__.__array_ufunc__` does exactly the same thing as `type(obj).__array_ufunc__`.  But since I use `Py_Type(obj)` in the C code, perhaps it is clearer to use `type(obj)` here too?